src/Repository.js

```diff
@@ -55,6 +55,9 @@ export default class Repository {
   }
 
   get packageConfigs() {
+    if (this.lernaJson.yarnWorkspaces) {
```

Comment:
This isn't sufficient because it doesn't pick up any potential `--use-workspaces` flag in the CLI args, only the "durable" config from a `lerna.json` (mostly equivalent to the relationship of `yarn` to a `.yarnrc` file, CLI flags always supersede the `lerna.json` values).